fdbserver/RestoreWorker.actor.cpp

```diff
@@ -80,8 +80,8 @@ ACTOR Future<Void> handlerTerminateWorkerRequest(RestoreSimpleRequest req, Refer
 
 // Assume only 1 role on a restore worker.
 // Future: Multiple roles in a restore worker
-ACTOR Future<Void> handleRecruitRoleRequest(RestoreRecruitRoleRequest req, Reference<RestoreWorkerData> self,
-                                            ActorCollection* actors, Database cx) {
+void handleRecruitRoleRequest(RestoreRecruitRoleRequest req, Reference<RestoreWorkerData> self,
+                                      ActorCollection* actors, Database cx) {
```

Comment:
The return statements in this function should also be updated to either remove them or change them to `return;`, as appropriate.